demos/999-storage-import-vhd-blob/azuredeploy.json

```diff
@@ -0,0 +1,169 @@
+                    }
+                },
+                "template": {
+                    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+                    "contentVersion": "1.0.0.210508",
+                    "parameters": {
+                        "location": {
+                            "type": "string",
+                            "defaultValue": "",
+                            "metadata": {
+                                "description": "The location in which the resources should be deployed."
+                            }
+                        },
+                        "Source": {
+                            "type": "string",
+                            "metadata": {
+                                "description": "URI of a Blob to fetch (Blob URI with SAS Token). For example: https://source-blob-storage-name.blob.core .windows.net/source-container-name/blob?sv=1234&spr=https&sig=1234"
+                            }
+                        },
+                        "Destination": {
+                            "type": "string",
+                            "metadata": {
+                                "description": "URI of a Storage Account container with a writable SAS Token. For example: https:// blob-storage-name.blob.core .windows.net/container-name?sp=racwl&st=2021-01-01T12:00:00&se=2021-02-01T12:00:00&spr=https&sv=2021-01-01&sig=123412341234"
+                            }
+                        },
+                        "deploymentScriptName": {
+                            "type": "string",
+                            "defaultValue": "ImportVHD-FromBlobUri",
+                            "metadata": {
+                                "description": "Name of the deployment script"
+                            }
+                        },
+                        "forceUpdateTag": {
+                            "type": "string",
+                            "defaultValue": "[newGuid()]",
+                            "metadata": {
+                                "description": "forceUpdateTag property, used to force the execution of the script resource when no other properties have changed."
+                            }
+                        }
+                    },
+                    "functions": [],
+                    "variables": {
+                        "uriVHDBlobSasToken": "[parameters('Source')]",
+                        "uriWritableStorageAccountBlobContainerSasToken": "[parameters('Destination')]",
+                        "userAssignedIdentityName": "[concat('ImportVHD-FromBlobUri-worker-',uniqueString(resourceGroup().id))]",
+                        "roleAssignmentName": "[guid(concat(variables('userAssignedIdentityName'),resourceGroup().id, 'contributor'))]",
+                        "forceUpdateTag": "[parameters('forceUpdateTag')]"
+                    },
+                    "resources": [
+                        {
```

Comment:
remove this resource - it doesn't appear to be needed (the deploymentScript does not use it)

Reply:
done